deploy/maps/action/configmap.action.deployment.nodejs.yaml

```diff
@@ -23,9 +23,9 @@ data:
     [
       { 
         "name":"appmetrics-dash", 
-        "text":"View AppMetric-Dash", 
+        "text":"View AppMetrics Dashboard", 
```

Comment:
Juni, please add a copyright statement to this file.  I know it was missing to begin with, but since we're updating it we should add it.

Reply:
Roger, this is the new file I created for this issue. Chris already approved the previous changes that I had merged but since you ask me to update the ui msg file, I also need to update this file again.  BTW the copyright is already there too. You may not expanded it. Thx.

Reply:
Reuse existing app metric msg instead.